cypress/support/hostsfunctions.ts

```diff
@@ -175,7 +200,34 @@ export function createAndEditAndDeleteHost(host_type: string, inventory: Invento
   const hostName = createAndCheckHost(host_type, inventory.name);
 
   // edit
-  editHost(inventory.id, host_type, hostName);
+  editHost(inventory.name, host_type, hostName);
   // delete
-  deleteHost(inventory.id, host_type, hostName);
+  deleteHost(inventory.name, host_type, hostName);
+}
+
+export function testHostBulkDelete(host_type: string, inventory: Inventory) {
+  //navigate to base screen
+  navigateToBaseView(host_type, inventory.name);
+  //create
+  const hostName1 = createAndCheckHost(host_type, inventory.name);
```

Comment:
Please try to use API calls using some support function, we dont want to create data manualy in tests, that are not particulary focused on test manual creation. Its for speedup the tests.

Reply:
that's the main goal of this - to check user UI actions